lib/web/desktop.go

```diff
@@ -392,6 +392,33 @@ func proxyWebsocketConn(ws *websocket.Conn, wds net.Conn) error {
 	return trace.NewAggregate(retErrs...)
 }
 
+// wasWebsocketClosedByClient handles the error returned by proxyWebsocketConn
```

Comment:
```suggestion
// handleProxyWebsocketConnErr handles the error returned by proxyWebsocketConn.
```

Handles the error how? Is it worth elaborating a bit?

Reply:
d9d1d6f87852a508b34da1da7665c013ff785feb